src/main/java/com/mastery/java/task/dto/validator/IsGenderValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.mastery.java.task.dto.validator;
+
+import com.mastery.java.task.dto.Gender;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class IsGenderValidator implements ConstraintValidator<IsGender, String> {
```

Comment:
what this class doing in the dto package?

Reply:
transfer it